requirements.txt

```diff
@@ -1,2 +1,3 @@
-requests==2.26.0
-click==8.0.1
\ No newline at end of file
+requests==2.23.0
```

Comment:
For some reason I had to use Python 2.7 to get that dependencies work. And pip showed confilcts for highter versions.